### ISSUE TITLE
[SCREENREADER]: Alt text for photo repeats the word image 

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#4255]
### [SCREENREADER]: Alt text for photo repeats the word image 

When a screen reader reads the alt text for the photo of the Pittsburgh VAMC University Drive Facility, the alt text is read as "Pittsburgh VAMC University Drive Facility image image". The screen reader says image again because it is an image element. This is a minor unpleasantry, but can feel like a thousand paper cuts for the person hearing it. If we can improve the experience for these users… The recommendation is to have the alt text read "Photo of Pittsburgh VAMC University Drive Facility", so that the screen reader reads as "Photo of Pittsburgh VAMC University Drive Facility, image". A screen reader user may be able to say to someone else, who may accompany them, that there is a photo of the facility. In this context, the detail that it is a photo adds useful context.

Each of the photos on this page would benefit the reader if context is provided.

## Type of Issue
Content

## Point of Contact

Jennifer

## Acceptance Criteria

As a screenreader user, I want to understand the context of images that may be helpful to me as I read the page content.

## Environment

* Operating System: all
* Browser: all
* Screenreading device: all
* Server destination: https://www.va.gov/pittsburgh-health-care/

## Steps to Recreate

1. Enter `https://www.va.gov/pittsburgh-health-care/` in browser
2. Start screenreading device listed in Environment
3. Navigate to the h1 above the duotone photo of the medical center
4. Read the content of the page starting at the h1
5. Verify the alt text for the photo

## Possible Fixes (optional)

**Current Code:**
```html
<img src="https://prod-va-gov-assets.s3-us-gov-west-1.amazonaws.com/img/styles/7_2_medium_thumbnail/public/2019-04/UD%20front.jpg" alt="Pittsburgh VAMC University Drive Facility Image" title="" width="100%">
```

**Recommended:**
```html
<img src="https://prod-va-gov-assets.s3-us-gov-west-1.amazonaws.com/img/styles/7_2_medium_thumbnail/public/2019-04/UD%20front.jpg" alt="Photo of Pittsburgh VAMC University Drive Facility" title="" width="100%">
```

## WCAG or Vendor Guidance (optional)

* [w3c H37: Using alt attributes on img elements](https://www.w3.org/TR/WCAG20-TECHS/H37.html)

* [Considerations when writing alt text](https://ux.shopify.com/considerations-when-writing-alt-text-a9c1985a8204)

**Comment by jenstrickland** [comment]
For visibility @SgardnerPMP  — this is the issue ticket I mentioned in our standup today, and we'll review tomorrow.

**Comment by davidconlon** [comment]
@SgardnerPMP FYI this is a content thing: https://staging.cms.va.gov/media/57/edit

@mmiddaugh Add to list with Jenny HL to discuss who should update PROD.

@1Copenut & @jenstrickland Question/Food for Thought: Should these images have null alt tags instead (and is that something we can do in drupal?)  They don't seem to add anything substantive to the page.

**Comment by jenstrickland** [comment]
@davidconlon — each of these photos would provide benefit to even non-sighted users, as they provide supplemental story and experience. If a non-sighted user has a support person, they would be able to tell the support person 'here's a photo of what where we're going looks like," and the acupuncture photo would benefit users through color/context of how a benefit may be used.

I inventoried each of these images:
1. Pittsburgh VAMC University Drive Facility duotone photo of the VAMC facility — adds useful context
2. The entrance to the Pittsburgh VA Medical Center--University Drive —  adds useful context 
3. Heinz campus of the VA hospital system in Pittsburgh —   adds useful context 
4. veteran meets stroke survivor and author — adds useful context, color
5. "Battlefield Accupuncture" — misspelled, and needs more context

**Comment by 1Copenut** [comment]
@davidconlon In cases where the images are decorative, you should be able to add `alt=""` or leave the Alt field empty in Drupal. I would ask that the content creator(s) review the HTML to make sure the empty alt tag was added correctly. `alt=""` is treated differently than no alt tag by screen readers, and signals different things to users.

In the case ^^ above, @jenstrickland makes strong points why the images should have rich alt text.

**Comment by mmiddaugh** [comment]
Notes from grooming: 
The blue image under the title is decorative only and is not necessary for user understanding. Use of a descriptive tag may actually cause greater confusion. We should use a null alt tag for this image so screenreaders skip the image entirely. 
In doing so, we need to understand any implication on the title tag and avoid inadvertently introducing an issue there.

**Comment by jenstrickland** [comment]
The blue image under the H1 provides information that is useful for non-sighted users — when it is an image of the facility, or perhaps an image of interaction between person being served and practicioner.

**Comment by ethanteague** [comment]
PR: https://github.com/department-of-veterans-affairs/vets-website/pull/12359
@mmiddaugh @jenstrickland , sounds like we want to keep alt text for these images. To that end, I've added a filter to the alt tag to prevent the word "image" from being added to said string at build time. This will prevent the word from being emoted twice, as screenreader appends the word "image" to alt tags associated with images.

**Comment by davidconlon** [comment]
Hang on.
@jenstrickland the blue image (and blue image alone on that page) is decorative **_and could be misleading_**.  I have conferred with designers on this and we can discuss further. 
The H1 of the page is "Pittsburgh Health Care" and _not_ University Drive.  These are two distinct, important differentiations.

If someone says "this is where we are going" on the blue photo that's false because the veteran could, in fact, be wanting to go to the HJ Heinz facility or one of the 6 clinics. 

Those are in the thumbnails below which _do_ provide the correct correlation and need correct Alt Tags.

**Comment by davidconlon** [comment]
Spoke with @jenstrickland 

Jennifer Strickland:spiral_calendar_pad:  11:49 AM
Michelle let me know that it was a system page — not a facility page — so the alt="" is fine, but I would also add role="presentation" for those.

11:49
Also  aria-hidden="true" .

**Comment by jenstrickland** [comment]
Thanks, Dave! Glad I understood the context!

`<img aria-hidden="true" role="presentation" alt="" … />` as a structure for guidance.

**Comment by mmiddaugh** [comment]
@jenstrickland  this one is ready for you

**Comment by jenstrickland** [comment]
Validated. 🎉
